e2etest/local_transfer_test.go

```diff
@@ -323,7 +323,7 @@ func TestLocalTransfer(t *testing.T) {
 	probeSvr := probe.New(7788)
 	require.NoError(probeSvr.Start(ctx))
 	defer func() {
-		require.NoError(probeSvr.Stop(ctx))
+		probeSvr.Stop(ctx)
```

Comment:
think we better find out the root-cause and fix, instead of just removing `NoError`?

it erred b/c `context canceled`, so call this before context cancelled is the right fix